### ISSUE TITLE
Will not build without giving mapbox a credit card number

### ISSUE THREAD

**Issue by another4til7** [basva923/GarminLive#2]
tl;dr: building requires a mapbox api/code which requires signing up for an account which requires a credit card

Building/Making the project in Android Studio (e.g. Build > Make Project) generates an error:

GET 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/mapboxsdk/mapbox-android-sdk/9.4.0/mapbox-android-sdk-9.4.0.pom'. Received status code 401 from server: Unauthorized

The only way to get authorized is to create a "free" account... which requires handing over a credit card number.... 

The build instructions indicate that the default app does not even show the mapbox.

Suggest modifying the code to not GET *api.mapbox.com* unless it is enabled.
